extensions/kubectl/command.go

```diff
@@ -96,7 +96,11 @@ func Command(client *rancher.Client, yamlContent *management.ImportClusterYamlIn
 		return "", err
 	}
 
-	steveClient := client.Steve
+	steveClient, err := client.Steve.ProxyDownstream(clusterID)
```

Comment:
`steveClient := client.Steve` is needed for local cluster. 

can you add in a condition where if the clusterID is local we set steveClient := client.Steve. if not, steveClient, err := client.Steve.ProxyDownstream(clusterID)